.github/workflows/maven.yml

```diff
@@ -11,19 +11,17 @@ on:
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
-
     steps:
-    - uses: actions/checkout@v2
-    - name: Set up JDK 1.8
-      uses: actions/setup-java@v1
-      with:
-        java-version: 1.8
-    - name: Build with Maven
-      run: mvn -B package --file pom.xml
-    - name: Archive snapshot
-      uses: actions/upload-artifact@v1
-      with:
-        name: kafka-connect-snmp-0.0.1-SNAPSHOT.jar
-        path: ./target/kafka-connect-snmp-0.0.1-SNAPSHOT.jar
+      - uses: actions/checkout@v2
+      - name: Set up JDK
+        uses: joschi/setup-jdk@v2
+        with:
+          java-version: '8'
```

Comment:
any particular reason we use 8 here and not 11? we have internally moved to 11 for our tools so just wondering the reasoning here.

Reply:
I had `UnsupportedClassVersionError`s including but not limited to `snmp4j` - I'll check Java 11 compat separately.